ReactWindows/ReactNative.Shared/UIManager/UIViewOperationQueue.cs

```diff
@@ -480,19 +481,16 @@ private void OnRendering(object sender, object e)
                 }
             }
 
+            Action[] actions = EMPTY_ACTIONS_ARRAY;
             lock (_gate)
             {
-                try
-                {
-                    foreach (var batch in _batches)
-                    {
-                        batch();
-                    }
-                }
-                finally
-                {
-                    _batches.Clear();
-                }
+                actions = _batches.ToArray();
```

Comment:
Curious though, it seems that in Android they don't use this approach for this component:
[https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java#L826-L833](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java#L826-L833)

---
In reply to: [88968974](https://github.com/ReactWindows/react-native-windows/pull/891#discussion_r88968974) [](ancestors = 88968974)